MekHQ/docs/DistributedMekHQ.md

```diff
@@ -0,0 +1,310 @@
+# Distributed MekHQ
+Distributed MekHQ is an idea to allow a slightly more sophisticated verison
+of "MekHQ by E-Mail". This document describes the design and architecture
+for Distributed MekHQ.
+
+## Objectives and Non-Goals
+This section describes the objectives of Distributed MekHQ and what will
+not be part of the design.
+
+### Objectives
+1. Support a common timeline between Campaigns.
+2. Support common scenarios and contracts between Campaigns.
+3. Support trading of parts, vehicles, and personnel between Campaigns.
+4. Support centralized contract, unit, and personnel markets.
+5. Support generation of OpFor from other Campaigns.
+
+### Non-Goals
+1. Distributed MekHQ will not support shared Campaigns.
+2. Distributed MekHQ will not support shared parts, vehicles, or personnel.
+3. Distributed MekHQ will not support controlling other Campaigns.
+4. Distributed MekHQ will not support chat.
+
+## Distributed MekHQ Design
+This section details the high level design of Distributed MekHQ and serves
+as the Use-Case basis for the architecture.
+
+### Concepts
+- **Host Campaign**: the Campaign which launched the distributed MekHQ session.
+- **Remote Campaign**: any Campaign not running as part of the local MekHQ client.
+- **Timeline**: the latest date among the connected Campaigns.
+- **Teams**: connected Campaigns which cannot be generated as OpFor in a contract.
```

Comment:
Do this mean each team will be tracked as if their own faction? And would this allow for campaign versus campaign battles?

Reply:
Less so Faction more "allies" w.r.t. OpFor generation. And yes, eventually PvP.